components/camel-workday/src/test/java/org/apache/camel/WorkdayComponentConfigurationTest.java

```diff
@@ -0,0 +1,75 @@
+package org.apache.camel;
+
+import org.apache.camel.component.workday.WorkdayComponent;
+import org.apache.camel.component.workday.WorkdayConfiguration;
+import org.apache.camel.component.workday.WorkdayEndpoint;
+import org.apache.camel.test.junit4.CamelTestSupport;
+import org.junit.Test;
+
+public class WorkdayComponentConfigurationTest extends CamelTestSupport {
```

Comment:
I am actually bit confused about naming. I'd go for something like WorkdayComponentTest` since you are testing the component itself